components/gitpod-protocol/src/typings/globals.ts

```diff
@@ -8,5 +8,6 @@ interface Window {
     gitpod: {
         service: import("../gitpod-service").GitpodService;
         ideService?: import("../ide-frontend-service").IDEFrontendService;
+        loggedUserID?: string;
```

Comment:
Should it be defined on the global for both dashboard and supervisor, when only supervisor will actually set the value?

Reply:
Define it in `gitpod-protocol` because [vscode browser will re-use it](https://github.com/gitpod-io/openvscode-server/blob/dadd40deda5959c0efdaa0c7d02ab64b6f8b5ad0/src/vs/gitpod/browser/workbench/workbench.ts#L7) @easyCZ

Reply:
Sure, perhaps drop in a comment on this field that it's used by VSCode. I think it's misleading as the definition is used by multiple apps, but only one of them actually sets the field.

Reply:
Updated 🙏